### DISCUSSION TITLE
Differential modifications with m6anet

### DISCUSSION THREAD

**Discussion by josiegleeson** [GoekeLab/m6anet#46] [General]
Hi,

I've seen in the latest update m6anet now outputs modification ratio of the reads. Can this be used to compare samples?

For example if I have two conditions, treatment1 and treatment2, and I want to compare m6a between transcripts from each condition. I also have replicates (3 v 3) for the conditions. Do you think its reasonable to compare the modifications between the two by either requiring probability modified >0.9 in one condition in every replicate vs probability modified <0.1 in the other, or would you suggest a more robust statistical test? 

Thanks for any advice.

**Comment by jonathangoeke** [comment]
Hi @josiegleeson, I would recommend to use [xPore](https://github.com/GoekeLab/xpore). If you are interested in m6A only, you can filter out any non A or non AC position. You could also run m6ANet in addition to xPore to identify m6A sites in the xPore output, that would probably be the most accurate approach. 

m6Anet will also give you sites that are modified, but not differentially modified, which could be of interest for example as a sanity check if the differences in m6A are small between conditions.

m6Anet also reutrns the stoichiometry which could be used to identify differential sites, but that is currently not tested, so xPore+m6Anet would be the best recommendation at the moment.
Thanks!
Jonathan

**Reply by josiegleeson** [reply]
Thanks Jonathan!

I think I'll just overlap with the m6anet sites for now and see.

Out of curiosity would you expect either the m6anet 'probability_modified' or 'mod_ratio' to correlate at all with the xpore 'mod_rate'? Or are these different measurements?
